gc/base/GCExtensionsBase.hpp

```diff
@@ -499,6 +499,7 @@ class MM_GCExtensionsBase : public MM_BaseVirtual {
 	};
 
 	HeapInitializationSplitHeapSection splitHeapSection; /**< Split Heap section to be requested */
+	bool enableScanConstantPoolObjectInScavenge; /**<  set true if need to scan ConstantPool Objects during Scavenge, the default is false (set true if Java 11 and above) */
```

Comment:
Why did you add it here but not in `MM_GCExtensions`? This is Java related content should not be in OMR
You don't need change OMR code at all

Reply:
Alternatively this flag might be put in `MM_ScavengerDelegate` so name can be reduced to `enableScanConstantPoolObject`. If we don't need command line option and can handle RootScannerReadBarrierVerivier (add ScavengerDelegate there) it is probably preferable solution